game-app/game-core/src/main/java/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -275,8 +276,8 @@ List<Unit> getUnits(final GameState data) {
     return t.getMatches(selectedUnits);
   }
 
-  public Territory getTerritory(GameData data) {
-    return data.getMap().getTerritoryOrNull(territoryName);
+  public @Nonnull Territory getTerritory(GameData data) {
```

Comment:
As discussed, this method is very sus.. Probably should not be used.
Consider deprecating it (for another PR):
```
    /**
     * @deprecated Use ... instead
     */
  @Deprecated
```